modules/imaging_browser/locale/imaging_browser.pot

```diff
@@ -29,3 +29,87 @@ msgstr ""
 
 msgid "New and pending imaging sessions"
 msgstr ""
+
+msgid "Visit QC Status"
+msgstr ""
+
+msgid "First Acquisition"
+msgstr ""
+
+msgid "First Insertion"
+msgstr ""
+
+msgid "Last QC"
+msgstr ""
+
+msgid "New Data"
+msgstr ""
+
+msgid "Links"
+msgstr ""
+
+msgid "SessionID"
+msgstr ""
+
+msgid "Sequence Type"
+msgstr ""
+
+msgid "Pending New"
+msgstr ""
+
+msgid "Entity Type"
```

Comment:
I think I mentioned on a different module's PR that this should be in the loris namespace